example_test.go

```diff
@@ -42,6 +45,9 @@ func ExampleGetMetricsSender() {
 	// Special case of count with value set to 1
 	metrics.Increment("my_counter")
 
+	// Gauge: records an absolute value
+	metrics.Gauge("my_gauge", 42)
```

Comment:
I just read the example file and I find this example confusing since you re-use the name of the package as a variable:
`metrics := metrics.GetMetricsSender()`

`metricsSender :=` ?